### ISSUE TITLE
Consider making pandablocks and bluesky visible components

### ISSUE THREAD

**Issue by stan-dot** [DiamondLightSource/developer-portal#713]
@coretl  I am not sure if that is needed, at the moment it is a bit unclear

**Comment by coretl** [comment]
@garryod I can't remember what we decided about shared orgs where DLS is a major contributor

The difficulty will be that we can't put catalog-info.yaml into any of the bluesky repos

**Comment by garryod** [comment]
If adding a `catalog-info.yaml` isn't an option then it might be best to create a GitLab repo with just a large `catalog-info.yaml` which points to the correct place, should hopefully be easy enough with the correct labels

**Comment by coretl** [comment]
I think that makes sense

**Comment by stan-dot** [comment]
I thought of that as well. @coretl could you please make the repo with just readme in a place that is fitting for it as a controls project? I will then scout out the Components to go into `catalog-info.yaml` into a PR to that new repo

then for bluesky the location for the corresponding repo is up to @callumforrester  and @coretl ?

**Comment by coretl** [comment]
I suggest you put all GitHub external references in a single repo.

I have just created https://gitlab.diamond.ac.uk/github/github-catalog-info for this. Scout away!

**Comment by stan-dot** [comment]
@garryod will this get automatically scraped by backstage once the branch is merged into main?

https://gitlab.diamond.ac.uk/github/github-catalog-info/-/tree/4-make-a-script-to-automate-this/bluesky

**Comment by garryod** [comment]
No, it needs to be at the root of the repo, and please tag me in the PR instead of an issue in another repo

**Comment by stan-dot** [comment]
the first part - ok, so I guess all three sources must end up in one long file.

the second part - I don't understand, @garryod

**Comment by garryod** [comment]
The appropriate place for discussion of the contents of that pull request is in that pull request, not here

**Comment by stan-dot** [comment]
https://github.com/DiamondLightSource/developer-portal/issues/713#issuecomment-2269509228


at the moment I considered this question to be more devportal related than related to that repo PR contents, as it is this repo (developer-portal) that has the code that does the scraping of the main branches of gitlab repos. fair enough
